data-processing/common/types.py

```diff
@@ -92,6 +94,7 @@ class Author:
 @dataclass(frozen=True)
 class Reference:
     s2_id: S2Id
+    corpus_id: S2CorpusId
```

Comment:
Wondering why the definition of this type isn't an optional string... Actually wondering the same thing about doi and arxiv_id...

Reply:
hmm yeah IDK... on L90 for `Author` they actually did make the authorId an `Optional`...

Does it not matter? confused...

Reply:
Hmmm that seems potentially more confusing than not having types at all. But also not your PR's problem.